tests/user_test.go

```diff
@@ -64,16 +66,17 @@ func TestUpdateUser(t *testing.T) {
 		CreateRequest:  userCreateRequest,
 		CreateResponse: CreateResponseUser{},
 		SubTests: []utils.SubTest{
-			{Name: "Success", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":"test2"}`}, ResponseCode: http.StatusOK, ResponseBodyRegex: `{"user":{.+}}`},
-			{Name: "Correctly updated", Request: utils.Request{Method: http.MethodGet, URL: userURL}, ResponseCode: http.StatusOK, ResponseBodyRegex: `{"username":"test2"}`},
-			{Name: "Empty Body", Request: utils.Request{Method: method, URL: userURL, Body: ``}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Empty json", Request: utils.Request{Method: method, URL: userURL, Body: `{}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Bad username key", Request: utils.Request{Method: method, URL: userURL, Body: `{"wrongkey":"test2"}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Empty username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":""}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Nil username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":nil}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Integer username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":1}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Object username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":{"somekey":"somevalue"}}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Not found", Request: utils.Request{Method: method, URL: userURL + "0", Body: `{"username":"test2"}`}, ResponseCode: http.StatusNotFound, ResponseBodyRegex: `{"error":"User not found"}`},
+			{Name: "Success", Request: utils.Request{Method: method, Body: `{"username":"test2"}`}, ResponseCode: http.StatusOK, ResponseBodyRegex: `{"user":{.*"username":"test2"}}`},
```

Comment:
Why did you add a regex .* before the username in the expected response? Are we currently returning another field ?

Reply:
No indeed, thanks to the `json:"-"` statement in models, this is not needed anymore.